task5/task5.rb

```diff
@@ -45,7 +36,21 @@ def initialize(file_name)
   def all
     @collection
   end
+
+  def movie_sort(field)
+    all.sort_by do |el|
+      return puts 'Wrong argument' unless
```

Comment:
1. We should raise error instead of printing it
2. We shouldn't check it everytime during iteration over all collection